docs/docs/contribute/creating-the-pr.mdx

```diff
@@ -0,0 +1,71 @@
+# Submitting your Local Changes
+
+> If you're new to contributing to open-source repositories using GitHub or find the instructions on this page confusing, we recommend signing up for one of our [Sharing is Caring events](https://pnp.github.io/sharing-is-caring/#pnp-sic-events). These events are completely free and designed to guide you through the process.
+
+Once you have everything created and your code/script sample is fully functional, you can proceed with creating a pull request (PR) to the CLI for Microsoft 365 repository to include your changes in the next release. Before submitting the PR, ensure that you have tested your code locally and that your tests provide 100% code coverage.
+
+:::info
+
+All git commands should be executed from the root of your locally cloned fork.
+
+:::
+
+If this is your first PR, make sure you have created a link to the upstream repository of the [CLI for Microsoft 365](https://github.com/pnp/cli-microsoft365).
+
+```bash
+# Check if you have a remote pointing to the CLI for Microsoft 365 repo:
+git remote -v
+
+# If you see a pair of remotes (fetch & pull) that point to https://github.com/pnp/cli-microsoft365
+# Then you are ok. Otherwise, you need to add one
+
+# Add a new remote named "upstream" and point it to the CLI repo
+git remote add upstream https://github.com/pnp/cli-microsoft365.git
+```
+
+## Check in your Local Changes
+
+Before proceeding, ensure that all your files are committed to your branch. If you're using Visual Studio Code, you can achieve this by selecting `Source Control` from the activity bar and providing a commit name in the `Source Control` tab. Generally, the commit name should be the title of the issue you are working on. Then click the commit button to safely commit your files to your local branch.
+
+If you're solely working with git, follow these steps:
+
+```bash
+# Stash all your files
+git add .
+
+# Commit your stash
+git commit -m 'Adds command spo group get'
```

Comment:
I think we should include the `.Closes #n` part of our commit message here. So that we reinforce that our contributors should be working from an issue.